tools/plugins/web-test-runner/vite.js

```diff
@@ -0,0 +1,63 @@
+// @ts-check - unfortunately a TypeScript plugin file is not supported
```

Comment:
It's quite possible this plugin and the `vite-node.js` runtime have enough overlap in responsibility that they should be combined.